appservice/src/deploy/runWithZipStream.ts

```diff
@@ -83,60 +82,52 @@ function getPathFromMap(realPath: string, pathfileMap?: Map<string, string>): st
     return pathfileMap?.get(realPath) || realPath;
 }
 
-const commonGlobSettings: Partial<globby.GlobbyOptions> = {
-    dot: true, // Include paths starting with '.'
-    followSymbolicLinks: true, // Follow symlinks to get all sub folders https://github.com/microsoft/vscode-azurefunctions/issues/1289
-};
-
 /**
  * Adds files using glob filtering
  */
-async function getFilesFromGlob(folderPath: string, site: ParsedSite): Promise<string[]> {
-    const zipDeployConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(ext.prefix, vscode.Uri.file(folderPath));
-    const globOptions = { cwd: folderPath, followSymbolicLinks: true, dot: true };
+export async function getFilesFromGlob(folderPath: string, resourceName: string): Promise<string[]> {
```

Comment:
A lot of this weird new logics is because a lot of these previous assumptions will break the `appservice` package when running the tests.